src/SFA.DAS.LevyTransferMatching.Web/Views/Pledges/ApplicationApprovalOptions.cshtml

```diff
@@ -0,0 +1,58 @@
+            </p>
+
+            <form method="post">
+                <div class="govuk-form-group" das-highlight-error-for="AutomaticApproval" error-class="govuk-form-group--error">
+                    <fieldset class="govuk-fieldset">
+                        <span das-validation-for="AutomaticApproval" class="govuk-error-message"></span>
+                        <div class="govuk-radios govuk-radios--conditional" data-module="govuk-radios">
+                            <div class="govuk-radios__item">
+                                <input class="govuk-radios__input" id="AutomaticApproval" name="AutomaticApproval" asp-for="AutomaticApproval" type="radio" value="false" das-highlight-error-for="AutomaticApproval" error-class="govuk-textarea--error" data-aria-controls="approval-content" />
+                                <label class="govuk-label govuk-radios__label" for="AutomaticApproval">
+                                    <strong>Check and review apprenticeship details</strong>
+                                </label>
+                                <ul class="govuk-list govuk-list--bullet">
+                                    <li>you may get multiple prompts if the employer adds the apprenticeship details in batches</li>
+                                    <li>you will be able to check the standard, the cost and dates</li>
+                                    <li>you will have the option to change the apprenticeship details or you can ask the employer to</li>
+                                </ul>
+                            </div>
+                            <div class="govuk-radios__item">
+                                <input class="govuk-radios__input" id="AutomaticApproval-2" name="AutomaticApproval" asp-for="AutomaticApproval" type="radio" value="true" das-highlight-error-for="AutomaticApproval" error-class="govuk-textarea--error" />
+                                <label class="govuk-label govuk-radios__label" for="AutomaticApproval-2">
+                                    <strong>Automatically approve the apprenticeship details</strong>
+                                </label>
+                                <ul class="govuk-list govuk-list--bullet">
+                                    <li>you will not check and review the apprenticeship details such as the apprenticeship standard, the cost or the start date</li>
+                                    <li>the employer or training provider can choose a different apprenticeship standard, but the cost of the apprenticeship details cannot exceed the agreed transfer amount in the application</li>
+                                </ul>
+                            </div>
+                        </div>
+                    </fieldset>
+                </div>
+                <div class="govuk-button-group">
+                    <button id="pledge-criteria-continue" class="govuk-button" data-module="govuk-button">
+                        Continue
```

Comment:
I noticed when I submitted the page without answering the question that I got a null reference exception.